app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Blukon.java

```diff
@@ -132,9 +138,9 @@ public static boolean checkBlukonPacket(byte[] buffer) {
                 UserError.Log.e(TAG, "Libre sensor has been removed!");
             }
 
-            m_gotOneTimeUnknownCmd = 0;
+            m_gotOneTimeUnknownCmd = false;
             currentCommand = "";
-            PersistentStore.setLong("blukon-4-hourly-scan", JoH.tsl()); // set to current time to force timer to be set back
+            PersistentStore.setLong(BLUKON_GETSENSORAGE_TIMER, JoH.tsl()); // set to current time to force timer to be set back
```

Comment:
I'm not exactly sure of the logic here but I think setting the persistent store may not work work. What is it that you're trying to achieve with this code?

Reply:
When getting an error, I would like to reset the timer, so that xdrip counts from "beginning".

What is the problem?

Reply:
Ok, I don't think that will work. Setting to the current time means the next time the rate limit would return true would be from the current timestamp. I'm assuming you want the opposite to happen so that it keeps trying to get the sensor time if the attempt fails?

Additionally there is a cache layer which will prevent setting the backend store directly from being seen by `pratelimit()`. When I described using the persistent store directly, that was an alternative to `pratelimit()` 

I think the best thing to do is for me to provide a helper method to reset the rate limit to avoid any confusion.

Reply:
see: https://github.com/jamorham/xDrip-plus/commit/ef2e8deead25616411e763d7d1053cf7d0fbb907 for the new helper